scripts/ErrorReporter/errorreport.ui

```diff
@@ -149,10 +152,34 @@ p, li { white-space: pre-wrap; }
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QPlainTextEdit" name="input_free_text">
+         <property name="frameShadow">
+          <enum>QFrame::Plain</enum>
+         </property>
+         <property name="plainText">
+          <string>Please enter any additional information about your problems. (Max 3200 characters)
```

Comment:
Can you/have you put a code limit on this 3200? I think you have put limits of 128 on the email/name boxes below?